modules/server_processes_manager/php/mriuploadserverprocess.class.inc

```diff
@@ -52,25 +52,25 @@ class MriUploadServerProcess extends AbstractServerProcess
     private $_sourceLocation;
 
     /**
-     * Root of the MRI files directory.
+     * The value of MRICodePath config setting.
```

Comment:
The changes to this file don't seem to be related to the API and should be moved to a different PR so someone more familiar with the server_processes_manager can review them.